classes/module/Module.php

```diff
@@ -3144,6 +3144,23 @@ protected function trans($id, array $parameters = array(), $domain = null, $loca
         $parameters['legacy'] = 'htmlspecialchars';
         return $this->getTranslator()->trans($id, $parameters, $domain, $locale);
     }
+
+    /**
+     * Check if the module uses the new translation system
+     * @return bool
+     */
+    public function isUsingNewTranslationsSystem()
```

Comment:
Please change
"isUsingNewTranslationsSystem"
to
"isUsingNewTranslationSystem"

🙂